docs/rfc/rfc-1.adoc

```diff
@@ -445,3 +445,45 @@ unlucky operator should submit a transaction on-chain to multiply the fee by a
 fixed amount (like 1.1x, 1.2x, etc up to some cap like 1.4x). With this in mind,
 we should only include deposits in the initial transaction whose `max_fee` is set
 high enough to withstand a fee increase up to whatever the maximum multiplier is.
+
+== Governable Parameters
+Alphabetized list of Governable Parameters with additional notes.
+
+
+- `bridge_deposit_flat_fee`: The flat fee for depositing bitcoin.
+- `bridge_deposit_percentage_fee`: The percentage fee for depositing bitcoin.
+- `dkg`: The minimum number of members we're allowed to drop down to during the
+  DKG group formation re-try period.
+- `dust_threshold`: The minimum bitcoin deposit amount for the transaction to
+  be included in a sweep.
+- `failed_heartbeat_reward_removal_period`: The amount of time an operator is
+  removed from reward eligibility after failing a heartbeat.
+- `group_size`: The total size of the signing group.
+- `heartbeat`: The number of group members required for a
+  <<heartbeat,heartbeat>> to be successful.
+- `heartbeat_block_length`: The number of ethereum blocks until the next heartbeat.
+  If set to 40, then the signers sign every 40th block.
+- `sweep_max_btc`: The amount of summed unswept bitcoin deposits that will
+  trigger an early sweep.
+- `sweep_period`: The amount of time we wait between sweeps.
```

Comment:
We can consider this change to make the definition more precise: 
```suggestion
- `sweep_period`: The amount of time we wait between scheduled sweeps on a wallet.
```

Reply:
resolved in 3829379